src/commands/deploy/setPreDeployTaskForDotnet.ts

```diff
@@ -0,0 +1,78 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import { window } from 'vscode';
+import { DialogResponses, IActionContext } from 'vscode-azureextensionui';
+import * as constants from '../../constants';
+import { cpUtils } from '../../utils/cpUtils';
+import { openUrl } from '../../utils/openUrl';
+import * as workspaceUtil from '../../utils/workspace';
+import { getWorkspaceSetting, updateWorkspaceSetting } from '../../vsCodeConfig/settings';
+import * as tasks from '../../vsCodeConfig/tasks';
+import { IDeployWizardContext } from "../createWebApp/setAppWizardContextDefault";
+
+export async function setPreDeployTaskForDotnet(context: IDeployWizardContext): Promise<void> {
+    await validateDotnetInstalled(context);
+    // follow the publish output patterns, but leave out tfw
+    const dotnetOutputPath: string = path.join('bin', 'Debug', 'publish');
```

Comment:
How were you able to get rid of all tfw logic?

Reply:
If we just rely on `dotnet publish`, we don't need to specify the framework. The output path is something that we are making ourselves.

In a multiple target framework scenario, the publish will fail telling the user to specify their twf